### ISSUE TITLE
Update `insert-ignores.sh` to current requirements

### ISSUE THREAD

**Issue by sjwarner-bp** [exercism/java#1450]
At the moment, the `insert-ignores.sh` script adds `@Ignore`, instead of `@Ignore("Remove to run test")`.

This message is part of our policy [to ignore non-initial tests](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests).

Additionally, the current script does not check if tests are already ignored. As such, it might be a good idea to determine if `org.junit.Ignore` is already imported, and if tests are already ignored. The script can then skip adding a second ignore or import. In this instance, the script produces something like:

```
 import org.junit.Test;
 import org.junit.Ignore;
 import org.junit.Ignore;
 
 import java.util.Arrays;
 import java.util.LinkedList;
 import java.util.List;
 
 import static org.junit.Assert.assertEquals;
 
 public class AccumulateTest {
 
     @Test
     public void emptyAccumulateProducesEmptyAccumulation() {
         List<Integer> input = new LinkedList<>();
         List<Integer> expectedOutput = new LinkedList<>();
         assertEquals(expectedOutput, Accumulate.accumulate(input, x -> x * x));
     }
 
     @Ignore("Remove to run test")
     @Ignore
     @Test
     public void accumulateSquares() {
         List<Integer> input = Arrays.asList(1, 2, 3);
         List<Integer> expectedOutput = Arrays.asList(1, 4, 9);
         assertEquals(expectedOutput, Accumulate.accumulate(input, x -> x * x));
     }
...
```

Note the duplication of both `import` and `@Ignore`.

**Comment by hgvanpariya** [comment]
@sjwarner-bp, I would like to take this if no one has already started ...

**Comment by sjwarner-bp** [comment]
Sure thing @hgvanpariya. Are the criteria clear enough? All we have to do is add the message to the inserted `@Ignore` and then some conditionals do detect if it the test is already Ignored

**Comment by hgvanpariya** [comment]
@sjwarner-bp , yes, got it .. I'll update it and then I'll need your help to review it ...

**Comment by sjwarner-bp** [comment]
Sounds good to me @hgvanpariya :+1:

**Comment by hgvanpariya** [comment]
@sjwarner-bp , so all the the `@Test` without `@Ignore` has to be tagged with `@Ignore("Remove to run test")` , correct ?

**Comment by sjwarner-bp** [comment]
That's correct!

**Comment by hgvanpariya** [comment]
@sjwarner-bp Update, please check and let me know if I missed anything ...

**Comment by hgvanpariya** [comment]
if there will be ` import org.junit.Ignore;` it won't add it again,
if there will be `@Ignore    @Test`  it won't add `@Ignore("Remove to run test")`

**Comment by sjwarner-bp** [comment]
Cool, I'll pull it down over lunch and test it out :slightly_smiling_face:

**Comment by sjwarner-bp** [comment]
Closed via #1455
